### DISCUSSION TITLE
Trimming - MATLAB - SFunction GUI

### DISCUSSION THREAD

**Discussion by hjorthchristian** [JSBSim-Team/jsbsim#1087] [Q&A]
Hello everyone,

I would like to use the JSBSim SFunction GUI, however it does not seem to be used anymore. Are there any trim control tools or similar that are used in collaboration with JSBSim with the use of MATLAB? I would like to trim the C-172 in with the GUI S Function or another tool in MATLAB but it does not seem possible with the current version of JSBSIM. 
I want to be able to have the same or similar functionality as the linearization manager provides in MATLAB! Any help is appreciated! :)

**Comment by bcoconni** [comment]
@agodemar I think some SFunction GUI was developed in your lab, wasn't it ?
